.github/workflows/deploy.yml

```diff
@@ -156,6 +155,111 @@ jobs:
           export DOCKER_IMAGE_TAG=testnet-${{ github.sha }}-$timestamp
           make deploy-daemons
           rm /home/runner/.docker/config.json
+  deploy-nano-testnet:
```

Comment:
suggestion: Maybe change this into a reusable workflow.

The [github reusable workflows](https://docs.github.com/en/actions/using-workflows/reusing-workflows#creating-a-reusable-workflow) allows us to create workflows in a separate file and just invoke it with some parameters, which would be less repetitive and easier to update in the future since updates would affect all deploy jobs.

Reply:
I did this in the base PR: https://github.com/HathorNetwork/hathor-explorer-service/pull/329

It seemed better indeed